### PR TITLE
Chores/optimize UI

### PR DESCRIPTION
**Pull request by oxSaturn**
Was using the site today with mobile Safari, and found some UI problems to be fixed.

## `filter: blur()` not working on Safari

Reproducible on Safari, both desktop and mobile:

Here's how it looks on Chrome/Edge:

## Move ConnectButton into dropdown

Users won't really need to connect/disconnect that frequently, so I decide to move the connect button into dropdown. Now we can have a much cleaner UI.

Here's the one before:

## Replace Dropdown with Drawer

We have a lot of content in that Dropdown, reckon it makes more sense using Drawer here as it could give us more space:

20230614b5DOJIjo@2x

---

## PR-Codex overview
This PR improves the user experience and fixes some bugs in the frontend code. 

### Detailed summary
- Updated styling for navigation and layout components
- Fixed Safari glitch with `transform` property
- Added `PageWrapper` component to handle conditional rendering based on user account status
- Improved typography and layout for several pages
- Refactored code for `Timer` component
- Removed unnecessary imports and code
- Minor bug fixes and improvements

> The following files were skipped due to too many changes: `Frontend-v1-Original/pages/liquidity/index.tsx`, `Frontend-v1-Original/pages/vote/index.tsx`, `Frontend-v1-Original/pages/vest/index.tsx`, `Frontend-v1-Original/components/header/mobileHeader.tsx`, `Frontend-v1-Original/public/images/logo-firebird.svg`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by t0rbik** [comment]
fyi this is still marked as draft so i dont look at it

**Comment by oxSaturn** [comment]
> fyi this is still marked as draft so i dont look at it

will change it to `ready for review` when ready. So far not yet.

File: Frontend-v1-Original/components/ssSwap/setup.tsx [hunk 1]
```diff
@@ -846,7 +846,7 @@ function Setup() {
         <div className="flex min-h-[176px] flex-col items-center justify-center">
           {renderSwapInformation()}
         </div>
-        <div className="mt-3 grid h-full w-full grid-cols-[1fr] gap-3 py-0">
+        <div className="mt-3 grid w-full grid-cols-[1fr] gap-3 py-0">
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/ssSwap/setup.tsx:849]: fix a bug

IMG_A0D6330ECA9F-1

**Comment by oxSaturn** [comment]
Chances are I can push more changes, but let me keep it small so it won't be pain in the ass to reivew.

**Review by t0rbik — APPROVED** [review]
This all looks good to me.

**Comment by t0rbik** [comment]
Any insights on overall page performance with our two blurred elems?

**Comment by oxSaturn** [comment]
> Any insights on overall page performance with our two blurred elems? Seems like doesnt do any harm

Seems fine, I don't see any significant performance problem. Most of time were spent on script, rendering and painting only a small portion:

20230615UpdAk8Jo@2x

**Pull request merged**